Automattic/wp-calypso
# Update/calypso signup complete event

#### Proposed Changes
**Add event props Request for calypso_signup_complete event**

This is from pau2Xa-4G5-p2

Addl enhancement for calypso_signup_complete event: Currently, we know whether a user has an item on their cart at the time of signup complete (eventprops = 'has_cart_items'); however we do not know the type of cart item and details of these cart items. 
Add
Service Slug
Product Slug
Product Variation Slug
details if there are any cart items at the time of signup?


#### Testing Instructions

- Go to [https://calypso.live/](https://calypso.live/?image=registry.a8c.com/calypso/app:build-56214)
- Ensure that[ Tracks Vigilante](https://github.com/Automattic/tracks-chrome-extension) is installed to monitor events 
- Click add a new site.
- Select a premium domain name 
- Select a paid package
- On the checkout page, look for the calypso_signup_complete event and ensure that the following props are present

1. domain_product_slug: ....  the value here will be dotblog_domain if a domain was purchased
2. plan_product_slug: ... the value here will change according to the plan selected, e.g., business-bundle for business bundles

- Repeat the process, but this time select "use a domain I own" and add any existing domain
- Select the "Connect your domain" option
- Select a hosting plan
- Ensure that the plan_product_slug prop is present and domain_product_slug is not there.

- Repeat the again, but this time select a Wordpress free domain
- Select a hosting plan
- Ensure that the plan_product_slug prop is present and domain_product_slug is not there.

- For the Free plan and Free domain, both the domain_product_slug and plan_product_slug won't be there. 

**Note:** The Service Slug WPCOM doesn’t know the service slug because it’s not exposed to the API.

Since this PR modifies the release build for editing-toolkit
 run install-plugin.sh editing-toolkit update/calypso_signup_complete_event on your sandbox and test the changes on WordPress.com.

**Screenshot of calypso_signup_complete event**

<img width="1253" alt="Screenshot 2023-02-06 at 07 53 00" src="https://user-images.githubusercontent.com/11347472/216892763-2fa90850-9981-45da-b143-33e1a439990c.png">




*

#### Pre-merge Checklist





Related to #
